### ISSUE TITLE
ps_linklist: impossible to order main items

### ISSUE THREAD

**Issue by Lionel-dev** [PrestaShop/PrestaShop#31306]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hello, 

When I try to change the positions of the main blocks, I can't do it. The icon however implies that it is possible.

![image](https://user-images.githubusercontent.com/48990561/218101995-bf546a07-c647-4696-95f3-3753a45ca5ee.png)

### Expected behavior

Ordering the main blocks

### Steps to reproduce

1. Install the module
2. Try to drag/drop the two basic items

### PrestaShop version(s) where the bug happened

1.7.8.8

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

ps_linklist

**Comment by djoelleuch** [comment]
Hello @Lionel-dev ,
Thanks for your report.
I was able to reproduce the issue with PrestaShop 8.0.0 and 1.7.8.8 .

[Untitled_ Feb 10, 2023 4_25 PM.webm](https://user-images.githubusercontent.com/115152460/218130850-feb87ac5-8118-4772-8eb9-40e887779ee4.webm)

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you.

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by Lionel-dev** [comment]
I will investigate this as soon as I have some time.

All I can say for now is that version 3.0.3 on a presta 1.7.6.4 does not have this bug. I install a more recent version and test

**Comment by Lionel-dev** [comment]
So here : 


```
<td class="position_handle-type column-position_handle clickable .js-drag-handle"> 
  <div class="position-drag-handle js-link_widget_grid_42-position" ...>
      <i class="material-icons">drag_indicator</i>
  </div>
</td>
```
            
If we remove the "js-drag-handle" point, the crosshair would reappear. Not having enough knowledge in the new prestashop modules (twig + symfony). I will have to learn before fixing. But maybe this point is the beginning of the bug :)

**Comment by PrestaEdit** [comment]
Hi, and thanks @Lionel-dev for the report !

@Devs : as I can see, there is a change into the GridExtension : https://github.com/PrestaShop/PrestaShop/commit/fe4415bf184ed6f13153fae25b0515f1ec79284d

It could be related ; a build of the ps_linklist "grid" JavaScript could maybe fix the issue, to be tested.

**Comment by florine2623** [comment]
The issue is fixed on 8.1.x, and 9.0.x

https://github.com/PrestaShop/PrestaShop/assets/16019289/10dc56cf-ee64-47ff-8b6e-202ac7f2a389

I'm closing the issue.
